iothub/device/src/Transport/Mqtt/MqttTransportHandler.cs

```diff
@@ -94,7 +94,7 @@ internal sealed class MqttTransportHandler : TransportHandler, IMqttIotHubEventH
         private const string ReceiveEventMessagePrefixPattern = "devices/{0}/modules/{1}/";
 
         private static readonly int s_generationPrefixLength = Guid.NewGuid().ToString().Length;
-        private static readonly Lazy<IEventLoopGroup> s_eventLoopGroup = new Lazy<IEventLoopGroup>(GetEventLoopGroup);
+        private readonly IEventLoopGroup _eventLoopGroup = GetEventLoopGroup();
```

Comment:
@drwill-ms, @timtay-microsoft, @azabbasi, @abhipsaMisra Does anyone know why this was originally static? Was it to set an upper bound of threads in the event of multiple clients/transports?

Reply:
Sorry, no idea. Frankly, I find the notion of a static thing in our SDK slightly frightening.

Reply:
Why not lazy anymore?